### PR TITLE
adds some flavour to the grandfather clock

### PR DESCRIPTION
**Pull request by press-mark**
## About The Pull Request
adds a quiet looping ticking sound to the grandfather clock & gives it the same time display as the new wall-mounted clocks.

currently the only place to find it is at reps office on blueshift (my beloved), central command and void raptor. maybe someone will map it into more places? ;)

## How This Contributes To The Skyrat Roleplay Experience

https://user-images.githubusercontent.com/6972764/193791696-8bc39009-aa6e-440f-b53b-5f4bbaa91737.mp4
no idea how to embed this properly (no ambient ship hum in this video, but it's pretty quiet)

i thought it would add a nice relaxing touch :)

## Changelog

:cl:
soundadd: the grandfather clock now ticks like a clock
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/officestuff/code/officestuff.dm [hunk 1]
```diff
@@ -3,6 +3,26 @@
...
+	mid_length = 12 SECONDS
+	volume = 10
+
+/obj/structure/awaymissions/wildwest/masterclock/Initialize(mapload)
+	. = ..()
+	soundloop = new(src, TRUE)
+
+/obj/structure/awaymissions/wildwest/masterclock/Destroy()
+	QDEL_NULL(soundloop)
+	. = ..()
```
Comment (Zonespace27) [inline modular_skyrat/modules/officestuff/code/officestuff.dm]:
```suggestion
	return ..()
```

File: modular_skyrat/modules/officestuff/code/officestuff.dm [hunk 2]
```diff
@@ -3,6 +3,26 @@
 	icon = 'modular_skyrat/modules/officestuff/icons/cowboyobh.dmi'
 	icon_state = "grandfather_clock"
 	desc = "Tick, tick, tick, tick. It stands tall and daunting, loudly and ominously ticking, yet the hands are stuck close to midnight, the closer you get, the louder a faint whisper becomes a scream, a plea, something, but whatever it is, it says 'I am the Master, and you will obey me.'"
+	var/datum/looping_sound/masterclock/soundloop
+
+// stolen from the wall clock
+/obj/structure/awaymissions/wildwest/masterclock/examine(mob/user)
```
Comment (Zonespace27) [inline modular_skyrat/modules/officestuff/code/officestuff.dm]: mind repathing this to something better? this isn't away mission stuff anymore

**Comment by Gandalf2k15** [comment]
is there a way to stop the ticking?

**Comment by press-mark** [comment]
not without destroying it, no

File: modular_skyrat/modules/officestuff/code/officestuff.dm (hunk 2 above)
Comment (press-mark) [reply modular_skyrat/modules/officestuff/code/officestuff.dm]: will do

while im at it, i may as well add it to the rest of the reps offices on other maps

**Review by tf-4 — COMMENTED** [review]
Add the ability to mute it with a multitool, I think it could get annoying if you can't do anything about it

**Comment by press-mark** [comment]
> Add the ability to mute it with a multitool, I think it could get annoying if you can't do anything about it

alright. you can now "screw the hands tight" to stop the ticking

**Review by tf-4 — APPROVED** [review]

**Pull request merged**